node/service/src/chain_spec/frequency_paseo.rs

```diff
@@ -54,7 +54,7 @@ fn template_session_keys(keys: AuraId) -> frequency_runtime::SessionKeys {
 pub fn local_paseo_testnet_config() -> ChainSpec {
 	// Give your base currency a unit name and decimal places
 	let properties =
-		get_properties(FREQUENCY_LOCAL_TOKEN, TOKEN_DECIMALS as u32, Ss58Prefix::get().into());
+		get_properties(FREQUENCY_TESTNET_TOKEN, TOKEN_DECIMALS as u32, Ss58Prefix::get().into());
```

Comment:
I set paseo-local to use `XRQCY`, so that it better matches with paseo live, but willing to move it back. However this aligns it for the metadata on all "paseo" chains.

Reply:
The variety of configs gets confusing sometimes--under what scenarios would we still use `FREQUENCY_LOCAL_TOKEN`, then?

Reply:
Ah, nvm. I see `paseo-local` !== `dev`